src/Events/RequestHandled.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Honeybadger\HoneybadgerLaravel\Events;
+
+use Illuminate\Foundation\Http\Events\RequestHandled as LaravalRequestHandled;
+
+class RequestHandled extends ApplicationEvent
+{
+    public string $handles = LaravalRequestHandled::class;
+
+    /**
+     * @param LaravalRequestHandled $event
+     * @return EventPayload
+     */
+    public function getEventPayload($event): EventPayload
+    {
+        $request = $event->request;
+        $response = $event->response;
+
+        $action = $request->route()->getActionName();
+        list($controller, $method) = explode('@', $action);
+
+        $metadata = [
+            'uri' => $request->url(),
+            'method' => $request->getMethod(),
+            'statusCode' => $response->getStatusCode(),
```

Comment:
@rabidpraxis re: naming conventions, I did some reading on conventions in the PHP community and it seems there is no clear consensus. I came across this [Laravel Best Practices page on github](https://github.com/alexeymezenin/laravel-best-practices?tab=readme-ov-file#follow-laravel-naming-conventions) which points to snake_case for object properties, but then I checked what kind of conventions I get with an example Laravel app and I see camelCase - so I went for that. Here is the `JobProcessed` event from the Laravel app I've been playing with:
```
{
  "uuid":"de1f0159-9665-4c5f-b4d6-620567790a06",
  "displayName":"App\\Jobs\\RandomJob",
  "job":"Illuminate\\Queue\\CallQueuedHandler@call",
  "maxTries":null,
  "maxExceptions":null,
  "failOnTimeout":false,
  "backoff":null,
  "timeout":null,
  "retryUntil":null,
  "data": {"commandName":"App\\Jobs\\RandomJob","command":"O:18:\"App\\Jobs\\RandomJob\":0:{}"}
}
```